website/docs/r/cm_catalog.html.markdown

```diff
@@ -1,39 +1,110 @@
 ---
-subcategory: "Catalog Management"
 layout: "ibm"
-page_title: "IBM : cm_catalog"
+page_title: "IBM : ibm_cm_catalog"
 description: |-
   Manages cm_catalog.
+subcategory: "Catalog Management API"
 ---
 
 # ibm_cm_catalog
 
-Create, modify, or delete an `cm_catalog` resources. You can manage the settings for all catalogs across your account. For more information, about managing catalog, refer to [catalog management settings](https://cloud.ibm.com/docs/account?topic=account-account-getting-started).
+Provides a resource for cm_catalog. This allows cm_catalog to be created, updated and deleted.
 
+## Example Usage
 
-## Example usage
-
-```terraform
+```hcl
```

Comment:
The example is just empty. It should be a valid example

Reply:
added example here